go.mod

```diff
@@ -6,27 +6,29 @@ require (
 	github.com/containernetworking/plugins v1.1.1
 	github.com/coreos/go-iptables v0.6.0
 	github.com/docker/docker v20.10.16+incompatible
-	github.com/golang/protobuf v1.5.2
 	github.com/moby/ipvs v1.0.1
 	github.com/onsi/ginkgo v1.16.5
 	github.com/onsi/gomega v1.19.0
-	github.com/osrg/gobgp v0.0.0-20211201041502-6248c576b118
+	github.com/osrg/gobgp/v3 v3.2.0
 	github.com/prometheus/client_golang v1.12.2
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.1
 	github.com/vishvananda/netlink v1.1.1-0.20210330154013-f5de75959ad5
 	github.com/vishvananda/netns v0.0.0-20211101163701-50045581ed74
 	golang.org/x/net v0.0.0-20220526153639-5463443f8c37
 	google.golang.org/grpc v1.46.2
-	k8s.io/api v0.23.7
-	k8s.io/apimachinery v0.23.7
-	k8s.io/client-go v0.23.7
+	google.golang.org/protobuf v1.27.1
+	k8s.io/api v0.24.1
```

Comment:
Do we want to upgrade to 0.24.x here? If so, we should also update the cri-api dependency.

Reply:
This has been updated.